### PR TITLE
Undo the update to 8.2.3 due to incompatibility for ZADE

### PR DESCRIPTION
**Pull request by CB-RyanMcCormick**
Revert "[HB-5631] Digital Turbine 8.2.3 (#36)"

This reverts commit 0bb5d59971f2ae33f7a4b1bc9a7c882490e8ee5a.

[HB-5631]: https://chartboost.atlassian.net/browse/HB-5631?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,7 +13,7 @@ The Chartboost Mediation Digital Turbine Exchange adapter mediates Digital Turbi
 
 In your `build.gradle`, add the following entry:
 ```
-    implementation "com.chartboost:chartboost-mediation-adapter-digital-turbine-exchange:4.8.2.3.0"
+    implementation "com.chartboost:chartboost-mediation-adapter-digital-turbine-exchange:4.8.2.1.1"
```
Comment (chauduyphanvu) [inline README.md:16]: 4.8.2.2.1?

Comment (CB-RyanMcCormick) [reply README.md:16]: Copy/Paste, but forgot to edit 🤦

Comment (chauduyphanvu) [reply README.md:16]: 4.8.2.2.1 or 4.8.2.1.1?

Comment (CB-RyanMcCormick) [reply README.md:16]: Jesus. How did I mess that up again

Comment (cb-jpadilla) [reply README.md:16]: All good. 😄

Comment (CB-RyanMcCormick) [reply README.md:16]: Wait.. wth. It shows 4.8.2.2.1 locally

**Review by cb-jpadilla — APPROVED** [review]
lgtm

File: README.md (hunk 1 above)
Comment (CB-RyanMcCormick) [reply README.md]: I guess I hadn't pushed my change 🤨 SourceTree showed everything as pushed though. Weird

**Review by chauduyphanvu — APPROVED** [review]

**Pull request merged**